Symfony/CS/Fixer.php

```diff
@@ -117,6 +125,26 @@ public function getConfigs()
     }
 
     /**
+     * Get ErrorsManager instance.
```

Comment:
If you refer to an object name its logical to use the casing of the class-name right?